tests/prune.test.mjs

```diff
@@ -46,6 +47,27 @@ describe('ARNS Record Pruning', () => {
     sharedMemory = transferMemory;
   });
 
+  it('messages with more than 100 bytes of data will throw an error', async () => {
+    const longMessage = 'a'.repeat(101);
+    const result = await handle({
+      options: {
+        Data: longMessage,
+        Tags: [
+          {
+            name: 'Action',
+            value: 'Info',
+          },
+        ],
+      },
+      memory: sharedMemory,
+    });
```

Comment:
nit: move to helpers.mjs

Reply:
[bf39b18](https://github.com/ar-io/ar-io-network-process/pull/381/commits/bf39b1855ac711d24a7312de02110971a3eb01b9)